bindgen/Main.cpp

```diff
@@ -62,6 +66,11 @@ int main(int argc, const char *argv[]) {
     std::string resolved = getRealPath(op.getSourcePathList()[0].c_str());
     LocationManager locationManager(resolved);
 
+    auto reuseBindingsConfig = ReuseBindingsConfig.getValue();
+    if (!reuseBindingsConfig.empty()) {
+        locationManager.loadConfig(reuseBindingsConfig);
```

Comment:
Any thoughts on also providing a built-in default config to ease use of the standard library bindings from Scala Native

Reply:
It may be a separate option, for example, `useStandardScalaNativeBindings: Boolean`

I looked through `native` package and found types only in `stdio` and in `complex` (but bindgen fails to translate `float complex`, `double complex` etc types, so currently it's not possible to reuse `complex`).

The easiest solution would be to hard code standard config in C++ code and "activate" it by passing `use-standard-scala-native-bindings`.

Or it is possible to support multiple config files, so sbt plugin can pass to bindgen a user-defined config(s) and a standard config

Reply:
I created an issue #147